frontends/web/src/locales/en/app.json

```diff
@@ -763,6 +763,10 @@
       }
     },
     "send": {
+      "change": {
+        "text": "The change will be returned to a Native Segwit address, unless you have at least one Taproot UTXO. In that case the change will be returned to a Taproot address.\n\nPlease note, if you are using coin control and select only Native Segwit UTXOs, the change will be returned to a Native Segwit address, regardless of whether you have a Taproot UTXO in that account. However, if you select Wrapped Swegit UTXOs, the change will still be returned to a Native Segwit address.",
```

Comment:
typo: Wrapped Swegit -> Wrapped Segwit

@jadzeidan this feels very convoluted, is there no easier and shorter way? How about:

"The change will be returned to a Taproot address if you have at least one Taproot UTXO.  If you use coin control, the change will be returned to a Taproot address if there is at least one Taproot UTXO among the selected UTXOS. In all other cases, the change is returned to a Native Segwit address."

Reply:
Sure, fine with me.

Reply:
fixed and pushed